cards/zgoo.json

```diff
@@ -0,0 +1,680 @@
+    "text": "Yuki Yagami deck only.\n[reaction] When your turn begins: Add 1 [bless] token to the chaos bag.\n[reaction] After your turn ends: Search the chaos bag for 1 [curse] token and return it to the token pool.",
+    "traits": "Avatar. Blessed.",
+    "type_code": "asset",
+    "version": 6
+  },
+  {
+    "code": "zgoo_00003",
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "flavor": "Many who undergo the shrine mained 'training' are unable to speak of it.",
+    "illustrator": "Audrius Mitkus",
+    "is_unique": true,
+    "name": "The Succession",
+    "pack_code": "zgoo",
+    "position": 7,
+    "quantity": 1,
+    "restrictions": "investigator:zgoo_00001",
+    "subtype_code": "weakness",
+    "slot": "body",
+    "subname": "That August Night",
+    "text": "<b>Revelation</b> - Put Baron Samedi into play and seal all [bless] tokens in the chaos bag on it.\n<b>Forced</b> - When a [bless] token would be added tot he chaos bag: Seal it on this and add a [curse] token to the chaos bag, instead.\n[action]: Test [will] (4). if you succeed, release all tokens sealed on The Succession and discard it.",
+    "traits": "Avatar.",
+    "type_code": "asset"
+  },
+  {
+    "back_flavor": "Makoto has amassed a small fortune renting out her services as a 'consort' to Arkham's well-paying elite. Though her expertise in massage therapy and exotic musical instruments are considerable, she is most recommended by far for her insight as a counselor and empathic confidant. But her uncanny ability to see into the minds of the rich and powerful and divine their hearts' true desires is no mere parlor trick; when her grandmother passed away, Makoto inherited a number of family heirlooms, including a strange ornamental headdress… The first time she tried it on, Makoto discovered that she was suddenly capable of perceiving a level of reality that lies hidden to most. So many long nights she has spent gazing into that abyss that it has begun to gaze back into her, as well...",
+    "back_text": "<b>Deck Size</b>: 30.\n<b>Deckbuilding Options</b>: Seeker cards ([seeker]) level 0-5, Survivor cards ([survivor]) level 0-1, <b>Patron</b> cards level 0-5, <b>Gambit</b> cards level 0-5, Neutral cards level 0-5.\n<b>Deckbuilding Requirements</b> (do not count toward deck size): The Infinite Oculus, Porcelain Mirage, 1 random basic weakness.\n<b>Deckbuilding restrictions</b>: No non-[[Patron]] assets that take up an ally slot.",
+    "code": "zgoo_00004",
+    "deck_limit": 1,
+    "deck_options": [
+      {
+        "faction": [
+          "seeker"
+        ],
+        "level": {
+          "min": 0,
+          "max": 5
+        }
+      },
+      {
+        "faction": [
+          "survivor"
+        ],
+        "level": {
+          "min": 0,
+          "max": 1
+        }
+      },
+      {
+        "faction": [
+          "neutral"
+        ],
+        "level": {
+          "min": 0,
+          "max": 5
+        }
+      },
+      {
```

Comment:
So this one is tricky.  I think we want to say "trait": ["patron"] as the first rule, then make a not: true, slot: ["ally"] rule second, then the standard ones.